### PR TITLE
feat (identity): Remove Web Account Manager

### PR DESCRIPTION
**Pull request by ferantivero**
closed: #573632

## WHY

we want to upgrade to the latest MSAL experience from the `Contoso.App` sample.

## WHAT Changed?

- reference Microsoft.Identity.Client 4 
- remove previous UWP identity implementation ([Web Account Manager](https://docs.microsoft.com/en-us/windows/uwp/security/web-account-manager))
- implement new MSAL experience
- implement sign-out flow. 

## HOW to Test?

You can clone this repository, configure your a Azure AD Application from Constants, and sign-in/out successfully using OAuth 2.0 client credentials flow on top of the Microsoft Identity platform.

### CODE REVIEW

File: ContosoApp/ViewModels/AuthenticationViewModel.cs [hunk 1]
```diff
@@ -328,35 +329,34 @@ public async void LogoutClick()
...
+                ApplicationData.Current.RoamingSettings.Values["IsLoggedIn"] = false;
+                SetVisibleAsync(vm => vm.ShowWelcome);
             }
         }
 
         /// <summary>
         /// Shows one part of the login UI sequence and hides all the others.
         /// </summary>
-        private async Task SetVisibleAsync(Expression<Func<AuthenticationViewModel, bool>> selector)
+        private void SetVisibleAsync(Expression<Func<AuthenticationViewModel, bool>> selector)
```
Comment (josevarela) [inline ContosoApp/ViewModels/AuthenticationViewModel.cs:347]: Let's change the name if the method is not async anymore

File: README.md [hunk 2]
```diff
@@ -21,6 +21,31 @@ A Universal Windows Platform (UWP) sample app that showcases UI controls (includ
...
 ![ContosoApp screenshot 1](images/customer-list.png)
 
+## May 2022 update
+
+This update includes:
+
+* Updated Plaform
+  * [WinUI](https://docs.microsoft.com/windows/apps/winui/winui2/getting-started) 2.6.1 to 3
+
+* Updated Target framwork
```
Comment (josevarela) [inline README.md:31]: splelling framwork => framework

**Review by josevarela — COMMENTED** [review]
Looks good. Two very small comments below

File: README.md (hunk 2 above)
Comment (ferantivero) [reply README.md]: good catch @josevarela 

done | addressed from 7a1a05595db129147d7701f3db88720953168d3a

**Pull request merged**